tao_compiler/mlir/disc/tests/tensorflow_ops/sqrt.cc

```diff
@@ -0,0 +1,70 @@
+/* Copyright 2021 The TensorFlow Authors. All Rights Reserved.
+
```

Comment:
Do we want to change the copyright part as this part of the code is provided by us.